harvester/metadata/tests/test_harvest_study_vocabulary.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import patch
+from datetime import datetime
+
+from django.test import TestCase, override_settings
+from django.core.management import call_command, CommandError
+from django.contrib.auth.models import User
+from metadata.models import MetadataField, MetadataValue
+
+
+@override_settings(VERSION="0.0.1")
+class TestHarvestStudyVocabulary(TestCase):
+    """
+    This test case represents the scenario where a harvest is started from t=0
+    """
+
+    fixtures = ["initial-study-vocabulary-resources"]
+    spec_set = None
+    repository = None
+
+    def setUp(self):
+        super().setUp()
+        self.user = User.objects.create(username="supersurf")
+        self.client.force_login(self.user)
+
+    def test_no_duplicate_error(self):
+        call_command("harvest_study_vocabulary", "--vocabulary=verpleegkunde")
+        call_command("harvest_study_vocabulary", "--vocabulary=verpleegkunde")
+
+    def test_same_number_applied_science(self):
+        with self.assertNumQueries(1032):
```

Comment:
Let's try to rework this together at some point. I might not have given the right ideas when fleshing this out. Sorry. Code is still useful though!